server.js

```diff
@@ -23,10 +23,27 @@ app.set('view engine', 'handlebars');
 //Body parser middleware
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({extended:false}));
+/* 
```

Comment:
Can you add the website link? like - 
Reference - 
1. abc.com

It will help us, to tract the references.

Reply:
connectFlash
1. https://www.npmjs.com/package/connect-flash